Feature user profile frontend

sahithipinnam opened the pull request:
user profile front end

apriilbee reviewed (changes requested):
Please address comments and only keep relevant changes to user profile frontend.

sahithipinnam commented:
All requested changes are completed:
- Reverted unrelated meal planner files
- Cleaned up profileController structure
- PR now contains only user profile frontend changes

Please review again when you're ready. Thanks!

apriilbee commented:
Fix merge conflict.

spacee48 commented on public/css/style.css:
Please pull the latest style.css from release-sprint-1-mvp and merge to your branch.

apriilbee commented on public/css/style.css:
```diff
@@ -1,291 +1,305 @@
+  }
+}
+
+.info-field p {
```
Are `.info-field p` and `.success-msg` the only css changes you did in `style.css`? If so, could you move  into `profile.css` and remove all the changes in `style.css`?

sahithipinnam commented:
All merge conflicts are resolved
Profile styles moved to profile.css
style.css and all unrelated files are clean

Let me know if anything else is needed..
